lib/mrb/mrb_logger.c

```diff
@@ -70,6 +81,7 @@ grn_mrb_logger_init(grn_ctx *ctx)
 
   mrb_define_method(mrb, klass, "need_log?", logger_need_log_p, MRB_ARGS_REQ(1));
   mrb_define_method(mrb, klass, "log", logger_log, MRB_ARGS_REQ(5));
+  mrb_define_method(mrb, klass, "path=", logger_path, MRB_ARGS_REQ(1));
```

Comment:
It should be `Groonga::Logger.default_path=` instead of `Groonga::Logger#path=`.